includes/classes/blocks/class-registration.php

```
@@ -59,6 +59,7 @@ public function enqueue_block_variations_script() {
 		$additional_scripts = [
```

PHPMD (2.15.0)

The variable $additional_scripts is not named in camelCase. (undefined)

(CamelCaseVariableName)